pkg/hash/v1alpha1/hash_test.go

```diff
@@ -0,0 +1,77 @@
+package hash
+
+import (
+	"testing"
+)
+
+func TestCalculateHash(t *testing.T) {
```

Comment:
Please break the this test into multiple test functions. Avoid using pool & disk names. Please provide generic names.

Reply:
i.e should I break it into functions like
1)TestHashList
2)TestHashStruct
3)TestHashInnerStruct
(or)
1)TestHashOne
2)TestHashTwo...
Which naming conventions are good?